crates/cheatnet/src/runtime_extensions/common.rs

```diff
@@ -23,3 +25,16 @@ pub fn sum_syscall_counters(mut a: SyscallCounter, b: &SyscallCounter) -> Syscal
     }
     a
 }
+
+#[must_use]
+pub fn get_relocated_vm_trace(vm: &VirtualMachine) -> Vec<TraceEntry> {
+    vm.get_relocated_trace()
+        .unwrap()
+        .iter()
+        .map(|x| TraceEntry {
+            pc: x.pc,
+            ap: x.ap,
+            fp: x.fp,
+        })
+        .collect()
```

Comment:
This can be quite expensive too (millions of elements potentially), maybe we should collect it conditionally? Sadly `get_relocated_trace` returns a reference to a vector :/